A small update about zh-CN

InkSsskeleton opened the pull request:
As you see,here's just a small update.It's a little embrassing that I forgot how to change my old Pull Request，and I'm failed to understand it once more,so this is the reason......Well,never mind about it.
By the way,how's the update going?Will we see the new version this year?

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton committed:
Update zh_CN.lang

Liahim85 commented:
> By the way, how's the update going? Will we see the new version this year?

Hello [InkSsskeleton](https://github.com/InkSsskeleton)!
Thank you for your help in translating and for the Chinese Wikipedia, which you write so carefully )
I sometimes look at it )
The mod is currently being ported to version 1.18. However, in parallel with this, I am rewriting many mechanics almost from scratch. Therefore, the process is not moving as fast as we would like.
Also, it makes no sense to follow the GitHub, since all changes occur in a closed repository.
However, you can follow the progress on the mod's official [discord](https://discord.gg/9qH4GQu).

Also, in light of current events, I've taken a short break from development. I hope to be able to return to work soon.

And here is a quote from the discord news channel:
> Not just the port:
>- **Terrain generator** - completely rewritten (no more generation lags! The edges of the cliffs now look as they were originally intended);
>- **Fog & clouds renderer** - completely rewritten (now it works entirely on its own shaders. No more translucent planes! Volumetric shadows and something else interesting 😉 );
>- **The growing trees system** - completely rewritten (now the generation and growth of trees can be controlled from the same places. This means that it will be easier for me to create many new tree-like vegetation)...
>... the work continues ...

InkSsskeleton commented:
> > By the way, how's the update going? Will we see the new version this year?
> 
> Hello [InkSsskeleton](https://github.com/InkSsskeleton)! Thank you for your help in translating and for the Chinese Wikipedia, which you write so carefully ) I sometimes look at it ) The mod is currently being ported to version 1.18. However, in parallel with this, I am rewriting many mechanics almost from scratch. Therefore, the process is not moving as fast as we would like. Also, it makes no sense to follow the GitHub, since all changes occur in a closed repository. However, you can follow the progress on the mod's official [discord](https://discord.gg/9qH4GQu).
> 
> Also, in light of current events, I've taken a short break from development. I hope to be able to return to work soon.
> 
> And here is a quote from the discord news channel:
> 
> > Not just the port:
> > 
> > * **Terrain generator** - completely rewritten (no more generation lags! The edges of the cliffs now look as they were originally intended);
> > * **Fog & clouds renderer** - completely rewritten (now it works entirely on its own shaders. No more translucent planes! Volumetric shadows and something else interesting wink );
> > * **The growing trees system** - completely rewritten (now the generation and growth of trees can be controlled from the same places. This means that it will be easier for me to create many new tree-like vegetation)...
> >   ... the work continues ...

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton committed:
Update zh_CN.lang

Added translations about config,command (thanks to Flame_Tree) and mobs(better than nothing after all)

InkSsskeleton committed:
Update zh_CN.lang

Added translations about config , command and mobs(thanks to Flame_Tree)

InkSsskeleton reviewed (commented):
I though I didn't succeed posting the previous file before. That's why I said it twice.

InkSsskeleton commented:
> I though I didn't succeed posting the previous file before. That's why I said it twice.

InkSsskeleton commented:
> I though I didn't succeed posting the previous file before. That's why I said it twice.

Dame you're just right here!Really embarrassing.

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton commented:
@Liahim85 ，Hello again!
I'm sorry that I have to send such massive words here instead of better websites,but for some reason,I'm unable to use Discord and many other foreign websites these days.Somehow,Github is accessable,so please forgive.
It's been 1 year at least since my last contribution to the Misty World mod,for I was busy studing in the high school.And now,since i've already graduated from high school,I guess it's time to deal with those,well,"issues left over from history".

In China ,there's a well-known standard of translation("信,达,雅",which is means "faithfulness, expressiveness and taste").Though we can just take some characters with similar pronunciation to the target word as a translation,and we actually did so,it's likely to be regarded as a low-quality translation and embrassing.So if someone need to translate a word into Chinese,it is quite important to know its meaning, its (word) roots,its animal prototype.Many mob names in the mod make me confusing and cannot figure out what they are(in both names and outlooks).So,could you please try to explain their names in your convenience?I'll list all the mob names needed to explain in the end.
If some of those listed name was just a flash in your mind,as long as you can remember,just tell,even if it's ridiculous.There's a mob called Andid in The Betweenland mod,which is the variant of "an idea".The mod's Chinese translaters at last translate it to "念鲤","念"has the meaning of "ideas",and"鲤" means carp,from its outlook.Even those experienced translaters need such informations,for someone not good at English and know almost nothing about foregin cultures,it only gets more important.

By the way,I also got some little questions to ask:
Why players cannot use lingering potion while wearing an unopen respirator?is it a bug or a setting on purpose?
Will the Taming skill be more easy to be realized in the next version?There's only 1 kind of Ⅰ level tamable mob,you know,it really needed to be optimized.
Could you please add some prefix like "foggy" to blocks that has the same name with something in vanilla Minecraft?It's quite weired and sometimes inconvenient.

I would be appreciated it if you could have a early and detailed reply.


The list:
Barvog
Wulder
Horb
Monk(mainly its animal prototype)
Galaga
Hulter
Barchiodon(mainly its root)
Momo(is it something like tapir?)

Liahim85 commented:
Hello!
Yes, of course, I will be happy to answer.
I myself haven’t worked on mod for a long time, so I’m glad that at least someone is busy with it )
It so happens that most of the names you listed mean nothing. I just liked them and thought they fit the mob.

For example, this applies to:
Barvog
Wulder
Horb
Galaga
Hulter
Momo

Hulter is a bit like a hunter.
Momo is, rather, a hybrid of a mini-hippopotamus with a platypus )
Barvog is like a barrel. But in reality it is a fat reptile.
Wulder - slightly similar to wool.

As for Barchiodon, it is rather a Greek name.
βραχίων [brachion] - shoulder. And the ending "don" is usually applied to various large animals. This gives a feeling of massiveness )

Horb - just went well with the mob )
Plus, it sounds similar to the Russian word for “hump.”

With "Monk" the situation is more complicated. I called him that precisely because he looks like a monk kneeling during prayer. He walks on “tucked” feet/hands. It's something like a sloth's claw. In addition, he has something like a bald patch on his head. In general, the following picture describes it fully )
![image](https://github.com/Liahim85/MistyWorld_Open/assets/17437986/ae334f5a-4139-4326-84c4-e62d30368210)

Liahim85 commented:
Now, on to other issues.
1. Not being able to use lingering potions is a bug. I'll fix it.
Although, I will need to think more about potions. We need to understand whether the player receives the effects from inhaling the potion or from contact with it?
2. The situation with taming is not as terrible as it seems. Yes, it may not be obvious at all, but at the beginning you can tame not only animals with level Ⅰ, but also with level ⅠⅠ. It is true that you can only get a tamed animal with the appropriate level. But you can also “try” to tame an animal 1 level higher than yours. You won't get any results, but you will still be leveling up. Not as fast as taming the right animal, but sooner or later, your level will rise and the process will go easier.
This can be done at any level of taming.
3. I'll think about it.

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton committed:
Update zh_CN.lang

InkSsskeleton commented:
>  Momo is, rather, a hybrid of a mini-hippopotamus with a platypus ) 

......sorry but,WTF?
I'll just keep the original name "莫貘“ then（means tapir yet also fit its pronounciation "Momo"),if nobody could come up with a better translation.
I've never taken it as the most abstract mob in this mod,until you just told me its prototype LOL

InkSsskeleton committed:
Update zh_CN.lang
